.devcontainer/localstack/init.sh

```
@@ -40,0 +1,39 @@
+                "HEAD",
+                "GET"
+            ],
+            "AllowedOrigins": [
+                "*"
+            ],
+            "ExposeHeaders": []
+        }
+    ]
+}'
+
+awslocal s3 mb s3://$bucketName
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)